### DISCUSSION TITLE
[BUG]Climax & pregnancy

### DISCUSSION THREAD

**Discussion by kyuubinotama** [ErinaSugino/Starbound-SBR-Mod#117] [Trouble Shooting] [answered]
Basicly climax and pregnancys wasnt working properly after a while dont know how it started or began, am using mods
[logfiles kyuubi.zip](https://github.com/user-attachments/files/15524241/logfiles.kyuubi.zip)

**Comment by ErinaSugino** [comment, marked as answer]
Going off of your latest log file included, you botched your pregnancy config.
`Caused by: (JsonParsingException) Error parsing json: unexpected character parsing word at 49:10`
